desktop/src/com/limegroup/gnutella/gui/menu/HelpMenu.java

```diff
@@ -54,9 +54,7 @@
 
         addSeparator();
 
-        addMenuItem(new OpenLinkAction("http://www.frostwire.com/stickers",I18n.tr("Order FrostWire Stickers"),I18n.tr("Order FrostWire stickers, we ship worldwide.")));
-
-        addMenuItem(new OpenLinkAction("http://www.frostwire.com/shop",I18n.tr("FrostWire T-Shirts"),I18n.tr("Get a T-shirt, support the project. You'll get also 10 free stickers")));
+        addMenuItem(new OpenLinkAction("http://www.frostwire.com/give",I18n.tr("Support Frostwire"),I18n.tr("Support Frostwire")));
```

Comment:
append this to the URL: `?from=desktop-help` ->
`http://www.frostwire.com/give?from=desktop-help`

Reply:
Frostwire has been misspelled, it's `FrostWire` (capital W)